source/get-started/compatibility-check.rst

```diff
@@ -61,8 +53,52 @@ Check compatibility
       SUCCESS: Supplemental Streaming SIMD Extensions 3 (ssse3)
       SUCCESS: Streaming SIMD Extension v4.1 (sse4_1)
       SUCCESS: Streaming SIMD Extensions v4.2 (sse4_2)
-      SUCCESS: Advanced Encryption Standard instruction set (aes)
       SUCCESS: Carry-less Multiplication extensions (pclmulqdq)
-      SUCCESS: EFI Firmware
 
-.. _clear-linux-check-config.sh: https://cdn.download.clearlinux.org/current/clear-linux-check-config.sh
+Option 2: Use a |CL| live image on a non-Linux system
```

Comment:
Option 1 works great. For Option 2, I need to test in appropriate environment; will do so later today.